Sources/MongoSwift/BSON/Document.swift

```diff
@@ -193,7 +195,7 @@ extension Document {
      */
     internal func get<T: BSONValue>(_ key: String) throws -> T {
         guard let value = self[key] as? T else {
```

Comment:
while you're here, the line above should use `getValue` to be safe, rather than subscript getter, since that could fatal error. can you update it?